ArduCopter/Log.cpp

```diff
@@ -67,7 +67,7 @@ void Copter::Log_Write_Control_Tuning()
         rangefinder_alt     : AP::logger().quiet_nanf(),
 #endif
         terr_alt            : terr_alt,
-        target_climb_rate   : target_climb_rate_cms,
+        target_climb_rate   : int16_t(target_climb_rate_ms * 100.0),
```

Comment:
this is fine as it's a non-functional change but I'd also be happy changing the logging so that it logs in meters.